src/components/button/templates/component/style.js

```diff
@@ -167,6 +182,14 @@ export let componentStyle = `
         .paypal-button .paypal-button-content .logo.logo-paypal {
             height: 17px;
         }
+        
+        .paypal-button.paypal-dual-true .paypal-button-content .logo.logo-paypal {
```

Comment:
OK this is fine, but we really have to start thinking about getting `less` in here... :D